examples/platforms/efr32/radio.c

```diff
@@ -770,15 +773,19 @@ void efr32RadioProcess(otInstance *aInstance)
     CORE_EXIT_CRITICAL();
 }
 
-void RAILCb_RfReady(void)
+void RAILCb_RxAckTimeout(void)
 {
+    if (sIsAckRequested)
```

Comment:
Yes, observing from testing between two dev boards (add log info at this callback), notice that for broadcast packet (e.g. advertisement), the log will be printed as well. so I add one check here only for unicast packet.